js-intermediate.js

```diff
@@ -93,6 +154,12 @@ var myCar = {
   }
 }
 
+let myDoors = "My car has " + myCar.specs.doors + " doors."
+let myTransmission = "My car has a " + myCar.specs.transmission + " transmission."
+
+console.log(myDoors)
+console.log(myTransmission)
+
```

Comment:
Again, nice work with the code, but with destructuring it could be shorter.

console.log(let { doors, transmission } = myCar.specs